site/src/api/api.ts

```diff
@@ -1006,7 +1006,7 @@ export const getWorkspaceBuildParameters = async (
   return response.data
 }
 type Claims = {
-  license_expires?: number
+  license_expires: number
```

Comment:
Is this type not returned from the typesGenerated.ts? Usually, the API endpoints and resources have generated types that are more trustable.